src/agg_bookend.c

```diff
@@ -34,24 +34,23 @@ TS_FUNCTION_INFO_V1(ts_bookend_deserializefunc);
 /* A  PolyDatum represents a polymorphic datum */
 typedef struct PolyDatum
 {
-	Oid			type_oid;
-	bool		is_null;
-	Datum		datum;
+	Oid type_oid;
```

Comment:
This still looks different. Spaces vs tabs?

Reply:
The difference here is that fields are not aligned instead one space is entered after a field. Clang does not support aligned declarations with the point right format. So we'd have to have unaligned fields. I think this is fine but if you disagree we can't use clang

Reply:
The difference here is that fields are not aligned instead one space is entered after a field. Clang does not support aligned declarations with the point right format. So we'd have to have unaligned fields. I think this is fine but if you disagree we can't use clang